provisioning/service/src/Exceptions/ProvisioningServiceClientException.cs

```diff
@@ -12,104 +11,41 @@ namespace Microsoft.Azure.Devices.Provisioning.Service
     /// <remarks>
     /// <code>
     /// ProvisioningServiceClientException
+    ///    |        \__IsTransient [identify if retry is a valid scenario]
     ///    |
     ///    +-->ProvisioningServiceClientTransportException [any transport layer exception]
     ///    |
-    ///    +-->ProvisioningServiceClientServiceException [any exception reported in the http response]
-    ///            |
-    ///            |
-    ///            +-->ProvisioningServiceClientBadUsageException [any http response 4xx]
-    ///            |        |
-    ///            |        +-->ProvisioningServiceClientBadFormatException [400]
-    ///            |        +-->ProvisioningServiceClientUnathorizedException [401]
-    ///            |        +-->ProvisioningServiceClientNotFoundException [404]
-    ///            |        +-->ProvisioningServiceClientPreconditionFailedException [412]
-    ///            |        +-->ProvisioningServiceClientTooManyRequestsException [429]
-    ///            |
-    ///            +-->ProvisioningServiceClientTransientException [any http response 5xx]
-    ///            |        |
-    ///            |        +-->ProvisioningServiceClientInternalServerErrorException [500]
-    ///            |
-    ///            +-->ProvisioningServiceClientUnknownException [any other http response >300, but not 4xx or 5xx]
+    ///    +-->ProvisioningServiceClientHttpException [any exception reported in the HTTP response]
     /// </code>
     /// </remarks>
-#if !WINDOWS_UWP
-    [Serializable]
-#endif
     public class ProvisioningServiceClientException : Exception
     {
-        private const string IsTransientValueSerializationStoreName = "ProvisioningException-IsTransient";
-        private const string TrackingIdSerializationStoreName = "ProvisioningException-TrackingId";
-        private const string ErrorCodeName = "ErrorCode";
-
         public bool IsTransient { get; private set; }
 
-        public string TrackingId { get; set; }
-
         public ProvisioningServiceClientException(string message)
-            : this(message, false)
-        {
-        }
-
-        public ProvisioningServiceClientException(string message, string trackingId)
-            : this(message, false, trackingId)
-        {
-        }
-
-        public ProvisioningServiceClientException(string message, bool isTransient, string trackingId)
-            : this(message, null, isTransient, trackingId)
+            : this(message, null, false)
```

Comment:
Nit: Same as above. Where the params are not easily understood it's nice to specify their names.

Reply:
done